iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/CertificateBasedAuthenticationCheckTest.java

```diff
@@ -57,10 +57,20 @@ void testRegistries() {
 
   @Test
   void testFactoriesLinkedServices() {
-    ArmVerifier.verify("CertificateBasedAuthenticationCheck/Microsoft.DataFactory_factories_linkedservices/test.json", new CertificateBasedAuthenticationCheck(),
+    ArmVerifier.verify("CertificateBasedAuthenticationCheck/Microsoft.DataFactory_factories_linkedservices/factories_linkedservices.json",
```

Comment:
Is it important that the test files are in the specific folders that are named after the resources? Or is it just done for our overview?

Reply:
I initially did it because I thought there were going to be potentially several files per resource, but in the end, there is only one :sweat_smile:
I will add a ticket to do a quick refactoring after all sub tickets are implemented to remove the layer folder, which is currently redundant.